### PR TITLE
New Website: Ensure Proper Use of Headers

### PR DESCRIPTION
**Pull request by andrew032011**
### Summary 
Ensure:
- Each page has a `<h1>` tag (always the hero section only)
- Don't skip over headers (i.e. every page now has `<h1>` for header, `<h2>` for subheader, and `<h3>` for headers below, and if there's a header below that, there's a `<h4>`. There aren't any usages of `<h5>`'s or `<h6>`'s anymore.

There should be no difference in the UI from this change.

### Notion/Figma Link 
Resolves problem 2 of https://github.com/cornell-dti/idol/issues/734.

https://www.notion.so/Address-Code-Audit-Comments-14b0ad723ce1802fa2f0c633511d8de3?pvs=4

### Test Plan 
Verify no UI changes are introduced by this change.

Grep for `<h1>`, `<h2>`, `<h3>`, `<h4>`, `<h5>`, `<h6>` and ensure that it meets the standards mentioned above.

### CODE REVIEW

File: new-dti-website/components/bottom.tsx [hunk 1]
```diff
@@ -35,9 +35,9 @@ const Bottom: React.FC = () => (
       <div className="text-left w-full md:self-center max-w-[520px] relative z-10">
         <div className="flex flex-col gap-y-1 pb-3 md:gap-y-2">
           <p className="text-neutral-200 text-xs font-bold uppercase lg:text-lg">courses</p>
-          <h5 className="text-lg font-semibold md:text-xl lg:text-[32px]">
+          <h3 className="text-lg font-semibold md:text-xl lg:text-[32px]">
```
Comment (Bookie0) [inline new-dti-website/components/bottom.tsx:38]:
```suggestion
          <h2 className="text-lg font-semibold md:text-xl lg:text-[32px]">
```


wait wdyt of making these ones `<h2>`


i'm looking at the homepage and it seems like these headers have the 2nd biggest font size on the page?
Screenshot 2024-12-01 at 12 32 52 PM

File: new-dti-website/components/bottom.tsx [hunk 2]
```diff
@@ -35,9 +35,9 @@ const Bottom: React.FC = () => (
...
             Teaching the Cornell Community
-          </h5>
+          </h3>
```
Comment (Bookie0) [inline new-dti-website/components/bottom.tsx:40]:
```suggestion
          </h2>
```

File: new-dti-website/components/bottom.tsx [hunk 3]
```diff
@@ -49,9 +49,9 @@ const Bottom: React.FC = () => (
               alt="Trends icon"
             />
             <div className="flex flex-col justify-start w-full items-start">
-              <h6 className="text-left w-full font-bold md:text-sm lg:text-[22px]">
+              <h4 className="text-left w-full font-bold md:text-sm lg:text-[22px]">
```
Comment (Bookie0) [inline new-dti-website/components/bottom.tsx:52]:
```suggestion
              <h3 className="text-left w-full font-bold md:text-sm lg:text-[22px]">
```

Then to not skip over anything we can bump the `<h4>`s to `<h3>`

File: new-dti-website/components/bottom.tsx [hunk 4]
```diff
@@ -49,9 +49,9 @@ const Bottom: React.FC = () => (
...
                 Trends in Web Development
-              </h6>
+              </h4>
```
Comment (Bookie0) [inline new-dti-website/components/bottom.tsx:54]:
```suggestion
              </h3>
```

File: new-dti-website/components/bottom.tsx [hunk 5]
```diff
@@ -108,9 +108,9 @@ const Bottom: React.FC = () => (
         <div className="flex flex-col gap-y-3 md:gap-y-4">
           <div className="flex flex-col gap-y-1 md:gap-y-2">
             <p className="text-neutral-200 text-xs font-bold uppercase lg:text-lg">outreach</p>
-            <h5 className="text-lg font-semibold md:text-xl lg:text-[32px]">
+            <h3 className="text-lg font-semibold md:text-xl lg:text-[32px]">
```
Comment (Bookie0) [inline new-dti-website/components/bottom.tsx:111]:
```suggestion
            <h2 className="text-lg font-semibold md:text-xl lg:text-[32px]">
```

same comment as this?

File: new-dti-website/components/bottom.tsx [hunk 6]
```diff
@@ -108,9 +108,9 @@ const Bottom: React.FC = () => (
...
               Expanding reach to our community
-            </h5>
+            </h3>
```
Comment (Bookie0) [inline new-dti-website/components/bottom.tsx:113]:
```suggestion
            </h2>
```

**Review by Bookie0 — APPROVED** [review]
nice! If you can, wanna take a stab at [these ones too](https://github.com/cornell-dti/idol/pull/755#issuecomment-2510133241)? :)

**Comment by Bookie0** [comment]
While we're at it, can we also use the appropriate headers on the other pages? Here are a few I noticed are missing:


## "Products" page

|Img|Code|
|-|-|
|Screenshot 2024-12-01 at 12 38 57 PM|Should be `<h2>`|
|Screenshot 2024-12-01 at 12 39 16 PM|Should be `<h3>`|
|Screenshot 2024-12-01 at 12 39 33 PM|Should be `<h3>`|
|Screenshot 2024-12-01 at 12 39 50 PM|Should be `<h2>`|


## "Course" page

|Img|Code|
|-|-|
|Screenshot 2024-12-01 at 12 41 11 PM|Should be `<h2>`|
|Screenshot 2024-12-01 at 12 41 27 PM|Should be `<h3>`|
|Screenshot 2024-12-01 at 12 41 46 PM|Should be `<h2>`|
etc.

**Review by JasonMun7 — APPROVED** [review]
This looks good to me!

**Pull request merged**